src/backend/api/reservations.js

```diff
@@ -0,0 +1,103 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//http://localhost:5000/api/reservations/contact_name
+router.get("/contact_name", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    const allReservations = await knex("reservation").select("contact_name");
+    response.json(allReservations);
+  } catch (error) {
+    throw error;
+  }
+});
+
+//-------------------------------------------
+
+// GET /api/reservations  - Returns all reservations
+router.get("/", async (req, res) => {
+  try {
+    const allReservations = await knex.select("*").from("reservation");
+    if (allReservations) {
+      res.status(200).json(allReservations);
+    } else {
+      res.status(404).send("The reservations were not found");
```

Comment:
Here we have the same case as with all meals - is it an error if we have no reservations or is it just a state that out app may have?